util/hypercall_external_signal/orchestrator-request.py

```diff
@@ -241,7 +249,14 @@ def main():
         "(auto-detected if not specified)",
     )
     parser.add_argument(
-        "function", choices=["status", "get_stats"], help="Function to execute"
+        "function",
+        choices=["status", "get_stats", "add_debug_flags"],
+        help="Function to execute",
+    )
+    parser.add_argument(
+        "--debug-flags-to-add",
+        metavar="FLAG1,FLAG2,..",
+        help="Sets the flags for debug output. This flag will only work if positional argument 'add_debug_flags' is passed",
```

Comment:
Again, make it clear that `-` means to remove it. Also, specify what happens if a flag is "invalid"